mopidy_musicbox_webclient/static/js/functionsvars.js

```diff
@@ -204,12 +204,14 @@ function getAlbum (pl) {
 function artistsToString (artists, max) {
     var result = ''
     max = max || 3
-    for (var i = 0; i < artists.length && i < max; i++) {
-        if (artists[i].name) {
-            if (i > 0) {
-                result += ', '
+    if (artists && artists.length > 0) {
```

Comment:
I'm sure we have code like this all over the place, doesn't JS have a .join() method?